localization/english/CWP/spain/francoist_reform_je_l_english.yml

```diff
@@ -0,0 +1,11 @@
+﻿l_english:
+francoist_reform: "Francoist Reform"
+francoist_reform_reason: "After the defeat of fascism in the war we need to change some fundamental things to get international recognition. It is time to slowly enforce some minimal reforms, we are not fascists, aren't we...?"
+francoist_reform_goal: "Our ideology was just based on Catholicism afterall. We will use Movimiento Nacional instead of the Party"
```

Comment:
```suggestion
﻿l_english:
francoist_reform: "Francoist Reform"
francoist_reform_reason: "With the defeat of fascism reforms must be made to garner international recognition. Because we're not fascists... #italic right?#!"
francoist_reform_goal: "Our ideology was just based on Catholicism after all. It's time for a #italic Movimiento Nacional!#!"
```